Update bootstrap documentation (#673)
digital-asset/docs.daml.com: docs/3.1/docs/canton/usermanual/identity_management.rst

~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> There are situations where a node should not be automatically initialized, but where we prefer to control

Reviewer: ```suggestion
There are situations where a node should not be automatically initialized, but where you should control
```

> each step of the initialization. For example, when a node in the setup does

Reviewer: ```suggestion
each step of the initialization. For example, this might be the case when a node in the setup does
```

not control its own identity, when you do not want to store the identity key on the node for security

> reasons, or when we want to set our own keys (e.g. when keys are externally stored in a Key Management Service - KMS).

Reviewer: ```suggestion
reasons, or when you want to set our own keys (e.g. when keys are externally stored in a Key Management Service - KMS).
```

> In the following, we demonstrate the basic steps of how to initialize a node:

Reviewer: ```suggestion
The following demonstrates the basic steps on how to initialize a node:
```
